### DISCUSSION TITLE
Is Keycloak affected by CVE-2023-4853

### DISCUSSION THREAD

**Discussion by tbroyer** [keycloak/keycloak#23295] [Q&A] [answered]
From https://quarkus.io/blog/cve-2023-4853/ it look likes that is not the case. Could someone confirm?

**Comment by vmuzikar** [comment]
Hi, Keycloak should not be affected but we're releasing 22.0.4 with fixed Quarkus version soon in any case.

CC @abstractj

**Comment by abstractj** [comment, marked as answer]
@tbroyer as far as I'm aware, Keycloak is not vulnerable to CVE-2023-4853. As @vmuzikar mentioned, the update should be available in the upcoming releases. 
The CVE affects only projects using path-based HTTP policy configuration, and Keycloak itself does not inherently use Quarkus for its HTTP security policies. I hope it helps.

**Reply by tbroyer** [reply]
That was my interpretation too, thanks for confirming.
